tests/visualization_tests/test_pareto_front.py

```diff
@@ -209,7 +209,7 @@ def test_plot_pareto_front_3d(
 
 
 @pytest.mark.parametrize("include_dominated_trials", [False, True])
-def test_plot_pareto_front_unsupported_dimensions(include_dominated_trials: bool) -> None:
+def test_plot_pareto_front_dimensions(include_dominated_trials: bool) -> None:
```

Comment:
Thank you for renaming the test function, but I feel this change is out of scope of this PR.
I think we have design choices of this test case. For example, we can also keep the current name, and remove the assertion of supported dimensions.
So, could you revert this change? 
```suggestion
def test_plot_pareto_front_unsupported_dimensions(include_dominated_trials: bool) -> None:
```

Reply:
@toshihikoyanase I agree with you. It will be another follow-up pull request.

@tohmae  Sorry for confusing you. I should have clarified my suggestion.